homework/max-of-vector/maxOfVector.hpp

```diff
@@ -3,6 +3,11 @@
 #include <limits>
 
 int maxOfVector(const std::vector<int>& vec) {
-    // TODO: Implement me :)
-    return {};
+   double hold = -2147483649;
```

Comment:
It’s better to use `std::numeric_limits<int>::min()`.
Your code will not work if the size of int is not equal to 4